docs/docs/user-guide/using-cli-context.md

```diff
@@ -0,0 +1,101 @@
+# Use CLI for Microsoft 365 context
+
+CLI for Microsoft 365 context provides unique functionality to save any kind of option and its value in a central place. The options saved in the context may be used in any kind of command execution, that way you may provide less or no options at all, which means using a context will help you save on keystrokes.
+It may be especially If you just want to separate and group all your parameters from your script in a single place.
+A context is saved in a m365rc.json file in your working directory. It can be committed to Source Control along with your script files. It can be managed by executing commands.
+
+## How to get started
+
+In order to create an empty context we may execute the following command:
+
+```powershell
+m365 context init
+```
+
+To add or update an option in the context use the `m365 context option set` command. The `name` is used to define the option name and `value` is used to define its default value. For example, if we want to set the option `listTitle` to `test list`, we should execute:
+
+```powershell
+m365 context option set --name 'listTitle' --value 'test list'
+```
+
+To remove a specific option from the context we can run:
+
+```powershell
+m365 context option remove --name "listTitle"
+```
+
+In order to remove the full context we may execute the following command:
+
+```powershell
+m365 context remove
+
+```
+
+## How does it work
+
+When a command is executed CLI will first check for the `.m365rc.json` file in the working directory. If present, the CLI will check if any of the options defined in it may be used for the currently executed command. If that's the case the CLI will execute this command with the option and its value taken from the context.
```

Comment:
```suggestion
When a command is executed, the CLI will first check for the `.m365rc.json` file in the working directory. If present, the CLI will check if any of the options defined in it may be used for the currently executed command. If that's the case the CLI will execute this command with the option and its value taken from the context.
```